UHResidentInformationAPI/V1/Domain/ResidentInformation.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using UHResidentInformationAPI.V1.Enums;
+
+namespace UHResidentInformationAPI.V1.Domain
+{
+    public class ResidentInformation
+    {
+        public string HouseReference { get; set; }
+        public int PersonNumber { get; set; }
+        public string TenancyReference { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string DateOfBirth { get; set; }
+        public string NINumber { get; set; }
+        public string UPRN { get; set; }
+        public List<Phone> PhoneNumber { get; set; }
+        public List<Email> Email { get; set; }
+        public Address Address { get; set; }
+    }
+
+    public class Address
+    {
+        public string AddressLine1 { get; set; }
+        public string AddressLine2 { get; set; }
+        public string AddressLine3 { get; set; }
+        public string AddressLine4 { get; set; }
```

Comment:
Did we clarify where these additional address lines come from in the UH database(s)?

I'll (re-)update the Swagger docs to match if we know for sure what data we'll be getting.

Reply:
I think I just copied over this address class from another repo. 
But thanks for highlighting this. 
Need to understand how the address is stored. 
The table has `address1` and `shortaddress` for a person.

Reply:
In the [Housing Information discovery document](https://docs.google.com/document/d/1uabFwcAgIc-y-uNI5nL4ccEdRkDcAn5CEpvJQ6Qo-e0/edit#heading=h.axlpnzqjpk2f), address for residents on the waiting list is covered, as well as address for next of kin. But the main subheading for address has no content. Perhaps addresses are more associated with tenancies/leaseholds, when it comes to the UH usecase/context. Let me post this in Slack also, to confirm.

Reply:
Thanks. I'll give this a read too.